063_cars_in_graphs/run.sh

```diff
@@ -0,0 +1,63 @@
+# make
+echo "====== running testcase1 (graph1, car1) ======"
```

Comment:
probably worth writing a simple loop to find the cross product of your test cases. This removes code duplication and also makes it clear that you are testing the cross product of your inputs ( or at least I'm assuming that).